src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+
+
+    public Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
You are creating more instances of 'ColorSupplier' class than you need. You should create only one instance of this class and use it for your purposes. Please, refer to the checklist.